config.go

```diff
@@ -111,6 +123,18 @@ func LoadConfig() (*Config, error) {
 		return nil, fmt.Errorf("error validating network: %v", err)
 	}
 
+	// Clean up and validate paths, then make sure the directories exist.
+	config.FaradayDir = lncfg.CleanAndExpandPath(config.FaradayDir)
+
+	// Append the network type to the data and log directory so they are
+	// "namespaced" per network.
+	config.FaradayDir = filepath.Join(config.FaradayDir, config.Network)
```

Comment:
Do we still want to namespace if this is a non-default dir?

Reply:
I don't have a strong opinion on it. My thought was to be as similar as possible to loop so users can expect the same behavior. But it would make sense to change it to non-namespaced directories in both projects?

Reply:
No let's leave it as is and keep it consistent across the two.